gateway-messages/src/sp_impl.rs

```diff
@@ -464,6 +469,10 @@ pub fn handle_message<H: SpHandler>(
             )
         }
         None => 0,
+        Some(OutgoingTrailingData::CabooseData(data)) => {
+            out[n..n + data.len()].copy_from_slice(data);
+            data.len()
+        }
```

Comment:
Ridiculously tiny nitpick - would you mind shifting the `None` to be the last case? :sweat_smile:

Reply:
Sure, done